packages/bun-types/scripts/bundle.ts

```diff
@@ -62,6 +62,31 @@ await write(
   JSON.stringify(packageJSON, null, 2) + "\n",
 );
 
+const tsConfig = {
+  compilerOptions: {
+    lib: ["ESNext"],
+    target: "ESNext",
+    module: "ESNext",
+    moduleResolution: "bundler",
+    moduleDetection: "force",
+    resolveJsonModule: true,
+    strict: true,
```

Comment:
`strict`? are u sure

Reply:
This is a best practice, we already have this in `tsconfig-for-init`

Reply:
Sorry to be clear

I mean for `bun init`

I think I'm confusing bun init and `bun-types`

Reply:
Strict mode is not for advanced users, it basically fixes a bunch of weird bugs in earlier versions of TS and they only added it as a flag for the sake of backwards compatibility. Everyone uses strict mode. That's not just me saying that as an "advanced" TS user. Zod requires strict mode to work and I don't get any complaints about it.